default/specials.txt

```diff
@@ -339,7 +339,7 @@ Special
             scope = Source
             activation = OwnedBy affiliation = AnyEmpire
             effects = [
-                SetDetection value = Value + 75
+                SetDetection value = Value + Min(Target.Construction / 30, 1) * 75
```

Comment:
I keep making this mistake as well, I constantly read that as parsing "make this the minimum" when it's actually "pick the lowest", it needs to be max(x,y) to have the desired effect. which is "pick the highest" not "make this the maximum", actually glad it's not just me getting that syntax wrong regularly.